staging/src/k8s.io/client-go/copy.sh

```diff
@@ -65,13 +65,13 @@ echo "move to the client repo"
 ls "${CLIENT_REPO}" | grep -v '_tmp' | xargs rm -r
 mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
 rm -r "${CLIENT_REPO_TEMP}"
-rm -r "${CLIENT_REPO}"/Godeps
 
 echo "moving vendor/k8s.io/kuberentes"
 cp -rn "${CLIENT_REPO}"/vendor/k8s.io/kubernetes/. "${CLIENT_REPO}"/
+rm -rf "${CLIENT_REPO}"/vendor/k8s.io/kubernetes
 # client-go will share the vendor of the main repo for now. When client-go
 # becomes a standalone repo, it will have its own vendor
-rm -r "${CLIENT_REPO}"/vendor
```

Comment:
I remember without the renaming e2e test will panic because there are two copies of glog and both try to register the `logdir` flag. I can double check.